CHANGES.md

```diff
@@ -15,6 +15,8 @@ Use list notation, and following prefixes:
 
 ### Next Release
 - Feature: Introduce internal logger (see [README/logger](/README.md#optional-confgiure-internal-logger) for configuration)
+- Change: `InPageMini` and `InPageStandard` controls now use `NotoSans` font instead of `NotoSansCJK` for Japanese and Korean languages
```

Comment:
It could be my English. Is `control` here a noun? If so, what does it mean?
```suggestion
- Change: `InPageMini` and `InPageStandard` now use `NotoSans` font instead of `NotoSansCJK` for Japanese and Korean languages
```

Reply:
Yes, `control` could be a noun and sometimes it's a synonym of `component` or a `widget`. For example: https://developer.apple.com/tutorials/swiftui/working-with-ui-controls

Reply:
I see. I usually call them UI components instead of UI controls.